DxDispatch/src/dxdispatch/Executor.cpp

```diff
@@ -469,8 +499,9 @@ Dispatchable::Bindings Executor::ResolveBindings(const Model::Bindings& modelBin
             source.elementOffset = modelSource.elementOffset;
             source.format = modelSource.format;
             source.resource = m_resources[modelSource.name].Get();
-            source.resourceDesc = &resourceDesc;
+            source.resourceDesc = (Model::ResourceDesc *) & resourceDesc;
```

Comment:
```suggestion
            source.resourceDesc = reinterpret_cast<Model::ResourceDesc*>(&resourceDesc);
```
consistency nit and avoiding C-style casts.